packages/components/src/system-message/components/deleted-message-content.tsx

```diff
@@ -0,0 +1,67 @@
+import { TimeoutIcon } from '@status-im/icons/12'
+import { DeleteIcon } from '@status-im/icons/20'
+import { Stack } from 'tamagui'
+
+import { IconAvatar } from '../../avatar'
+import { Button } from '../../button'
+import { Text } from '../../text'
+
+import type { SystemMessageState } from '../system-message'
+
+type Props = {
+  timestamp: string
+  text: string
+  action?: {
+    label: string
+    onClick: () => void
```

Comment:
Can you rename it to `onPress` to keep it consistent with RN terminology?

Reply:
Fixed in [`d094693` (#367)](https://github.com/status-im/status-web/pull/367/commits/d094693b68dcc26372faef25e1e87cd7f74d9820)